Library/Homebrew/software_spec.rb

```diff
@@ -336,20 +336,18 @@ def checksum_for(tag)
   end
 
   def checksums
-    checksums = {}
-    os_versions = collector.keys
-    os_versions.map! do |macos|
+    tags = collector.keys.sort_by do |tag|
       begin
-        MacOS::Version.from_symbol macos
+        MacOS::Version.from_symbol tag
       rescue
-        nil
+        "0.#{tag}"
```

Comment:
Ok, sorry, I see this is a `sort_by` and misread as a `map`. Could you add a comment to explain this above the rescue and then I think we're good to go.